source/docs/articles/sites/code/what-is-the-pantheon_api-module.md

```diff
@@ -14,6 +14,7 @@ The code for the API module is available within our upstream repositories:
 
 - [Drupal 6](https://github.com/pantheon-systems/drops-6/tree/master/modules/pantheon)
 - [Drupal 7](https://github.com/pantheon-systems/drops-7/tree/master/modules/pantheon)
+- [Drupal 8](https://github.com/pantheon-systems/drops-8/tree/master/modules/pantheon)
 
```

Comment:
pantheon-systems/drops-8/issues/19 

Suggest linking to the above issue and noting development